base/stream.jl

```diff
@@ -1264,6 +1264,11 @@ function (f::RedirectStdStream)(::DevNull)
     nulldev = @static Sys.iswindows() ? "NUL" : "/dev/null"
     handle = open(nulldev, write=f.writable)
     _redirect_io_libc(handle, f.unix_fd)
+    c_sym = f.unix_fd == 0 ? cglobal(:jl_uv_stdin, Ptr{Cvoid}) :
```

Comment:
Rather than duplicate, why not
```suggestion
f(handle)
close(handle)
return devnull
```
?

Reply:
I also had to think about it for a second, but I don't see any reason why it would be.